src/agent/onefuzz-task/Cargo.toml

```diff
@@ -85,4 +85,6 @@ schemars = { version = "0.8.12", features = ["uuid1"] }
 
 [dev-dependencies]
 pretty_assertions = "1.4"
+proptest = "1.2.0"
```

Comment:
Why not use the latest version? 1.3.1

Reply:
There's no reason, thanks for catching that.